src/popup/App.vue

```diff
@@ -14,16 +14,27 @@
         class="app-inner"
         :class="{ 'styled-scrollbar': showScrollbar }"
       >
-        <Header v-if="delayedShowHeader" />
+        <Header v-if="isAuthenticated && delayedShowHeader" />
 
-        <!-- We are disabling animations on FF because of a bug that causes flickering
-          see: https://github.com/ionic-team/ionic-framework/issues/26620 -->
+        <!--
+          We are disabling animations on FF because of a bug that causes flickering
+          see: https://github.com/ionic-team/ionic-framework/issues/26620
+        -->
         <IonRouterOutlet
+          v-if="isAuthenticated"
```

Comment:
`v-if="isAuthenticated"` is probably not sufficient, we need to also check if the wallet is initialized, opening the app for the very first time right now results in user being stuck here 
![image](https://github.com/user-attachments/assets/e86a924c-2ffd-4cd7-a010-f2dde72d05d0)

Reply:
The reason for that was that the `isAuthenticated` state was not updated to `true` after importing or creating new wallet. I fixed this, but in rather ugly way. It's ugly because the `isAuthenticated` state is stored in `auth` composable, but the setting password is in `accounts`, so the accounts is not able to update the `auth` state.

This is why I really think we need to move the authentication logic from `accounts` to `auth`.
@martinkaintas @smaroudasunicorn @CedrikNikita